SITCOMTN-023.tex

```diff
@@ -49,13 +48,129 @@
+\begin{table}
+\begin{center}
+\caption{\label{tab:owner} Summary of Ownership}
+\begin{tabular}{|p{1.8cm}|p{1.8cm}|p{1.8cm}|p{1.8cm}|p{1.8cm}|p{1.8cm}}
+\hline
+Ticket type vs Use Case              & Milestone 2	& Milestone 3& Epics	& Story/Bug	& Tasks \\
+\hline
+\hline
+Milestone Tracking & {\bf Creator}: P6 &{\bf Creator}: P6& {\bf Creator}: One SCLT Coordinator & {\bf Creator}: SIT-Com member & {\bf Creator}: Anyone \\
+ & {\bf Reviewer}: SITCom manager or delegate & {\bf Reviewer}: one SCLT lead or manager & {\bf Reviewer}: one SCLT lead & {\bf Reviewer}: chosen by the ticket creator or a SCLT coordinator & {\bf Reviewer}: chosen by the ticket creator or a SCLT coordinator \\
+\hline
+Planned activities & -- & -- & Same as above & Same as above & Same as above \\
+\hline
+Emergent issues & -- & -- & -- & Same as above & Same as above \\
+\hline
+\end{tabular}
+\end{center}
+\end{table}
+
+\subsection{Link to Other Jira Projects}
+This project needs to be integrated with the other already existing Jira projects. The following list covers the potential required links.
+
+\begin{itemize}
+\item Summit: this is the Jira project used to define the activities being conducted at the summit facilities everyday. These activities are ranked by priorities 1 to 4.  The SITCom tasks will appear on the summit calendar.
+\item DM: the DM project gathered all the activities (planned, bugs, etc) conducted by the software team at large. It includes the pipeline as well as the control software tasks. 
+\item IT: the IT project is reserved for all the activities conducted by the IT group, both at the summit, in Tucson and in La Serena. Similarly to the DM project, it covers both planned activities as well as emerging issues. 
+\item CAP: CAP stands for the Commissioning Activities Planning and is used for work coordination of software needed for commissioning. There was some discussion to extend this particular Jira project to match our needs, however, there was enough differences that we decided not to go that route. As a result, this project might be merge into the SITCom project as we move forward.  
+\item LVV: LSST Verification and Validation project gathers all the requirement verification and validation procedures as well as incremental testing procedures.
+\item Obs: the Observing Operation Jira project was created for night observation mostly, during which issues are encountered and need to be addressed within the next day or two. Its workflow is basic, allowing for quick reporting, requiring the observer to only add essential details (fault title and description, time of the fault, urgency). The triage and redirection to appropriate assignees and Jira projects is being done by a delegate of each technical group as soon as possible the next day.
+\item FRACAS: Failure Reporting, Analysis and Corrective Action System. 
+\item Risk: Risk management which we probably will not need to link to. 
```

Comment:
why is this here if we don't care about it?

Reply:
This shows we thought about all dependencies.